src/domain/entities/TransactionLimitedPartnership.ts

```diff
@@ -0,0 +1,11 @@
+// will be removed after these fields are added to the LimitedPartnership structure in the sdk
+
+import { LimitedPartnership } from "@companieshouse/api-sdk-node/dist/services/limited-partnerships";
+
+type TransactionLimitedPartnership = LimitedPartnership & {
+  _id?: string;
+  created_at?: Date;
+  updated_at?: Date;
```

Comment:
Does the web service need to know about these dates? They're just meta data the the API will update when it receives requests.

Reply:
I wasn't sure about these dates

Reply:
I would remove them if not needed.

Reply:
The entire file will be removed once the sdk has been updated